lapce-data/src/editor/commands/test/test_state.rs

```diff
@@ -73,14 +73,9 @@ impl TestState {
             let marker_len = end - start;
 
             if whole_match.as_str().starts_with("</") {
-                ends.insert(cursor_id, start).map(|_| {
-                    panic!("Duplicate selection end marker: {whole_match:?}")
-                });
-            } else {
-                starts
-                    .insert(cursor_id, start)
-                    .map(|_| panic!("Duplicate cursor marker: {whole_match:?}"));
-            }
+                if ends.insert(cursor_id, start).is_some() { panic!("Duplicate selection end marker: {whole_match:?}") }
+            } else if starts
+            .insert(cursor_id, start).is_some() { panic!("Duplicate cursor marker: {whole_match:?}") }
```

Comment:
Would you mind running `rustfmt` please? This looks a bit weird and I'm not sure if that's because of Github's diff ui, or if the code is formatted weird.

Reply:
Thanks, done. My fault to overlook this.